src/ja/worker/main.py

```diff
@@ -15,20 +15,28 @@ class JobWorker:
     """
     The main class on the Worker Client
     """
-    def __init__(self, config_path: str = "/etc/jobadder/worker.conf") -> None:
+    def __init__(
+            self, config_path: str = "/etc/jobadder/worker.conf",
+            socket_path: str = "/run/jobadder-worker.socket/",
```

Comment:
same thing as the other PR, why `/` at the end?

Reply:
Because I copy-pasted the typo.